keyboards/keebio/quefrency/keymaps/yoryer/keymap.c

```diff
@@ -0,0 +1,49 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```

Comment:
QMK does this automatically; you don't need this line.

```suggestion
```

Reply:
I've made the requested changes on keymap.c file